projects/packages/my-jetpack/_inc/components/jetpack-manage-banner/index.jsx

```diff
@@ -8,10 +8,12 @@ import jetpackManageIcon from './jetpack-manage.svg';
  * Jetpack Manager Banner component that renders a banner with CTAs.
  *
  * @param {object} props - Component props.
+ * @param {boolean} props.isAgencyAccount - Whether users account is an Agency account or not.
  * @returns {object} The JetpackManageBanner component.
  */
 const JetpackManageBanner = props => {
-	const { isAgencyAccount } = props;
+	// eslint-disable-next-line no-unused-vars
+	const { isAgencyAccount = false } = props;
```

Comment:
Is there a reason we are keeping this variable here if it's no longer being used?

Reply:
@CodeyGuyDylan, the only reason I left this variable here is mainly for visibility, because in about a week or so we will be changing the button again to say, "Sign up" and/or "Manage your account", and we will probably use/need this variable at that time.

Reply:
That makes sense, sounds good to leave it here 😄